### DISCUSSION TITLE
Median curve and bounds

### DISCUSSION THREAD

**Discussion by mhamilton-avenir** [neherlab/covid19_scenarios#800] [General]
Hi, I have a few very basic questions about how variance is handled.

* When plotting the number of infectious with upper and lower bounds, I notice the median typically exceeds the upper bound at the peak. Why is that? The median curve for cumulative deaths is always falls roughly in the middle of the bounds.
* From other threads, I see the upper and lower bounds reflect the 20th and 80th percentiles of samples of the parameter range. Is that sampling only from the range in R0, or also from other parameter ranges not shown on the dashboard?

**Comment by rneher** [comment]
Hi @mhamilton-avenir,

are you referring to the values shown in the graph, or exported values in a table. In the graph, I can't see any case where the median exceeds the upper bound. A trajectory corresponding to the median value can very well exceed the trajectory for the highest value at some point in time. 

We samples the ranges for R0 and the effectiveness of mitigations. Other parameters are fixed at a specific value. 

hope this helps,
richard

**Comment by mhamilton-avenir** [comment]
Thanks Richard. Attached are screen shots for Tanzania with default parameters. If you set linear axis and remove susceptibles and recovered, you can see the median instantaneous number infectious just slightly exceeds the range. This happens only at the peak, as you say, but it’s true for (almost?) every one of the ~100 scenarios I’ve run for several African countries. So it seems like there must be some simple explanation based on what it means to be the median sampled value.

The scenarios I’m making are not weird – no mitigation, just varying your fitted R0 ranges slightly by a factor. Cumulative deaths, perhaps because cumulative rather than instantaneous, are always right in the middle of the range.

Matt

**Reply by rneher** [reply]
Ok, I think I know why this is. Our terminology is a little confusing. We sample parameters from the ranges given for R0 and the mitigations and generate a bunch of trajectories. We then report the 20% and 80% percentiles at each time point for each quantity as upper and lower bounds. In addition, we run a trajectory for the average parameters. This mostly falls right in the middle of the bounds other than right at the peak (some parameter combinations peak earlier, others later, reversing the order of the lines). We opted for showing the deterministic center trajectory to produce reproducible output between runs, but we probably shouldn't call it "median". 

![image](https://user-images.githubusercontent.com/8379168/90404211-627e7580-e0a2-11ea-83e9-0446f58931a4.png)

hope this makes sense.

**Comment by mhamilton-avenir** [comment]
Okay, yes that does make sense. To restate, you run N trajectories, and the upper and lower bounds just string together the 20th and 80th percentiles of each value (S, I, R, etc.), taken independently for each quantity at each time point. The solid line is something different, a new trajectory run with the mean of the sampled parameter. It does not correspond to any string of points drawn from the N sampled trajectories, and in particular not to the 50th percentile.

We don’t see any of the other quantities, such as number susceptible or cumulative fatalities, exceed their bounds. That’s because for the number infectious (unlike other quantities) every curve curves peaks at a different time and must exceed all the others at its own peak. This new trajectory for number infectious just does the same, and because it’s the mean of parameters, it tends to peak in the middle of the others.

Thanks Richard, that clears it up!
